### PR TITLE
Re-License Vaultwarden to AGPLv3

### PR DESCRIPTION
**Pull request by BlackDex**
This commit prepares Vaultwarden for the Re-Licensing to AGPLv3
Solves #2450

### CODE REVIEW

**Review by williamdes — APPROVED** [review]

**Review by Gyarbij — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 [![Docker Pulls](https://img.shields.io/docker/pulls/vaultwarden/server.svg)](https://hub.docker.com/r/vaultwarden/server)
 [![Dependency Status](https://deps.rs/repo/github/dani-garcia/vaultwarden/status.svg)](https://deps.rs/repo/github/dani-garcia/vaultwarden)
 [![GitHub Release](https://img.shields.io/github/release/dani-garcia/vaultwarden.svg)](https://github.com/dani-garcia/vaultwarden/releases/latest)
-[![GPL-3.0 Licensed](https://img.shields.io/github/license/dani-garcia/vaultwarden.svg)](https://github.com/dani-garcia/vaultwarden/blob/master/LICENSE.txt)
+[![AGPL-3.0 Licensed](https://img.shields.io/github/license/dani-garcia/vaultwarden.svg)](https://github.com/dani-garcia/vaultwarden/blob/master/LICENSE.txt)
```
Comment (stefan0xC) [inline README.md:10]: The master branch was renamed to main.

```suggestion
[![AGPL-3.0 Licensed](https://img.shields.io/github/license/dani-garcia/vaultwarden.svg)](https://github.com/dani-garcia/vaultwarden/blob/main/LICENSE.txt)
```

Comment (BlackDex) [reply README.md:10]: Then this is also still faulty in the current README i think ;) and it then needs to be changed there within an other PR.

Comment (stefan0xC) [reply README.md:10]: Okay, will do. :+1:

**Review by GeekCornerGH — APPROVED** [review]

**Review by dani-garcia — APPROVED** [review]

**Comment by Ornias1993** [comment]
Did you guys actually consult a copyright lawyer?
Because you sure as hell did not have permission from all copyright holders (pr committers, basically) to relicense their works and the GPL doesn't grant you that right either.

While you legally could add GPL code into an AGPL project, the virality clause basically forced you to GPL the result as well, leading to an underground requirement to also respect people using the resulting software as GPL.

TLDR:
You now have GPL+AGPL duallicense, which basically completely nullified your AGPL license...

**Comment by tessus** [comment]
I suggest you read https://github.com/dani-garcia/vaultwarden/discussions/2450
